themes/navy/layout/partial/after_footer.swig

```diff
@@ -1,14 +1,14 @@
 <!-- Scripts -->
 <!-- Cookie -->
-<script src="https://cdnjs.cloudflare.com/ajax/libs/js-cookie/2.2.0/js.cookie.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/js-cookie@2/src/js.cookie.min.js"></script>
 <!-- build:js build/js/main.js -->
 {{ js('js/lang_select') }}
 {{ js('js/toc') }}
 {{ js('js/mobile_nav') }}
 <!-- endbuild -->
 {% if page.layout === 'plugins' %}
 <!-- Plugin search -->
-<script src="https://cdnjs.cloudflare.com/ajax/libs/lunr.js/2.3.6/lunr.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/lunr@2.3.6"></script>
```

Comment:
"https://cdn.jsdelivr.net/npm/lunr@2.3.6/lunr.min.js" or
"https://cdn.jsdelivr.net/npm/lunr@2/lunr.min.js" (whichever you prefer)